cmd/sbctl/verify.go

```diff
@@ -52,11 +58,16 @@ func VerifyOneFile(state *config.State, f string) error {
 	if err != nil {
 		return err
 	}
+
+	file_entry := &VerifiedFile{FileName: f, FileIsSigned: false}
```

Comment:
```suggestion
	fileentry := &VerifiedFile{FileName: f, IsSigned: false}
```

Prefer CamelCase in Go :) Note that `IsSigned` is initialized as "false", so you don't have to write it at this point.